mllib/src/main/scala/org/apache/spark/mllib/clustering/BisectingKMeans.scala

```diff
@@ -246,6 +245,16 @@ class BisectingKMeans private (
     new BisectingKMeansModel(root, this.distanceMeasure)
   }
 
+  /**
+   * Runs the bisecting k-means algorithm.
+   * @param input RDD of vectors
+   * @return model for the bisecting kmeans
+   */
+  @Since("1.6.0")
```

Comment:
Nit: this should be since 2.4?

Reply:
You couldn't call `BisectingKMeans.run(...)` before this, right? it wasn't in a superclass or anything. In that sense I think this method needs to be marked as new as of 2.4.0, right?

Reply:
`def run(input: RDD[Vector]): BisectingKMeansModel` is a public api since 1.6, and users can call it.

Reply:
Oh right I get it now, this isn't a new method, it's 'replacing' the definition above. 👍